packages/fiori/src/SideNavigationGroup.ts

```diff
@@ -0,0 +1,177 @@
+import property from "@ui5/webcomponents-base/dist/decorators/property.js";
+import litRender from "@ui5/webcomponents-base/dist/renderer/LitRenderer.js";
+import slot from "@ui5/webcomponents-base/dist/decorators/slot.js";
+import {
+	isLeft,
+	isRight,
+} from "@ui5/webcomponents-base/dist/Keys.js";
+import customElement from "@ui5/webcomponents-base/dist/decorators/customElement.js";
+import { getI18nBundle } from "@ui5/webcomponents-base/dist/i18nBundle.js";
+import type I18nBundle from "@ui5/webcomponents-base/dist/i18nBundle.js";
+import "@ui5/webcomponents-icons/dist/navigation-right-arrow.js";
+import "@ui5/webcomponents-icons/dist/navigation-down-arrow.js";
+import Icon from "@ui5/webcomponents/dist/Icon.js";
+import SideNavigationItemBase from "./SideNavigationItemBase.js";
+import type SideNavigationSelectableItemBase from "./SideNavigationSelectableItemBase.js";
+import type SideNavigationItem from "./SideNavigationItem.js";
+import SideNavigationGroupTemplate from "./generated/templates/SideNavigationGroupTemplate.lit.js";
+
+import {
+	SIDE_NAVIGATION_GROUP_HEADER_DESC,
+} from "./generated/i18n/i18n-defaults.js";
+
+// Styles
+import SideNavigationItemCss from "./generated/themes/SideNavigationItem.css.js";
+
+/**
+ * @class
+ *
+ * ### Overview
+ *
+ * Represents a group of navigation actions.
+ * The `ui5-side-navigation-group` can to be used inside a `ui5-side-navigation` only.
```

Comment:
Remove or if it's important to remain:
The `ui5-side-navigation-group` can only be used inside a `ui5-side-navigation`.

Reply:
Did the latter